support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -87,4 +87,25 @@ export const tests: Tests = {
     seed: 10,
     optimizeId: '-2DJ0JTsSVaskWwTm9Je4A',
   },
+  comparisonTableTest: {
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'variant',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 0,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    targetPage: pageUrlRegexes.subscriptions.digiSub.nonGiftLandingAndCheckout,
```

Comment:
This test is targeting all regions via the main digital subs landing page and checkout (as we are tracking conversions), and should not appear on the gift page or gift checkout.